src/pages/resultsView/oncoprint/TracksMenu.tsx

```diff
@@ -394,12 +426,23 @@ export default class TracksMenu extends React.Component<IAddTrackProps, {}> {
         },
     });
 
+    @action.bound
+    private onSelectGenericAssayProfileByType(
+        genericAssayType: string,
+        profileId: string
+    ) {
+        this.selectedGenericAssayProfileIdByType.set(
+            genericAssayType,
+            profileId
+        );
+    }
+
     @computed
     private get genericAssayTabs() {
         let tabs = [];
         if (this.isGenericAssayDataComplete && this.showGenericAssayTabs) {
```

Comment:
This is my fault, it looks like I forgot to add necessary checks in `isGenericAssayDataComplete` function.
Could you add `this.props.store.genericAssayEntitiesGroupByMolecularProfileId.isComplete` and `this.selectedGenericAssayProfileIdByType.size > 0` into `isGenericAssayDataComplete` function?